views/update/update-beneficial-owner-type.html

```diff
@@ -31,16 +31,16 @@
 {% set entity_name = entity_name %}
 {% set entity_number = entity_number %}
 
-{% if not errors %}
```

Comment:
I'm just wondering why this if not errors check is no longer required?

Reply:
I can’t determine why this `if not errors` is here in the first place, including it makes it so that the list of BO’s you’ve added already doesn’t show on the page after a validation error is triggered, but I can figure out why we would want that to disappear or how this could break the code

It's hard to understand from the commit where it was added

Reply:
Right, so you're only removing it in order to clean up the code? If you leave it, do tests all still pass and the app behave as expected?

Reply:
Hi, yes essentially to clean up the code. But I'd like to understand the purpose better before we move ahead. So the behaviour of the app on the current main branch if you trigger a validation error on this page is in this screenshot:

![image](https://github.com/companieshouse/overseas-entities-web/assets/164036596/8d9798b0-373f-4334-8996-d37957e7dad4)

I assumed that the summary table at the top should not disappear, so deleted `{% if not errors %}` in the template, but it's hard for me to determine from the original PR's why this was included in the first place. If i'm right Toro Loco where not directly involved with this page, do we have anyone we can ask about this?

Reply:
To answer your question, unit tests still pass and I'd like to add an additional test to make sure this content "What you have reviewed" and "What you have added in this update" is still displayed even if errors are present on the page. Checking with our BA also about what the expected behaviour ought to be

Reply:
Yes, Update journey implemented by a Kainos team originally, so no history on this unfortunately. As discussed, as long as it's well tested, this change is hopefully fine and correct.